api-tests/features/cimit/alternate-doc.feature

```diff
@@ -27,6 +27,9 @@ Feature: CIMIT - Alternate doc
     Then I get an OAuth response
     When I use the OAuth response to get my identity
     Then I get a 'P2' identity
+    And the TICF VC has properties
+      | cis  |                              |
+      | type | RiskAssessment               |
```

Comment:
I think we can probably omit this, now that the step above checks for the presence of a TICF VC.

So that in most scenarios we simply confirm that we have called TICF and returned the VC, and in the TICF-specific scenarios, we can check that it contains appropriate content.